server/datasources.local.js

```diff
@@ -0,0 +1,22 @@
+var debug = require('debug')('MeshModels.server.datasource');
+var url = require('url');
+
+var dbUrl = url.parse(process.env.STRONGLOOP_MESH_DB || 'memory://');
```

Comment:
Would this ever be something other than a "DB", like a web service of some kind?